sandbox/cgroup/tracking.go

```diff
@@ -20,6 +20,13 @@ var cgroupProcessPathInTrackingCgroup = ProcessPathInTrackingCgroup
 
 var ErrCannotTrackProcess = errors.New("cannot track application process")
 
+// TrackingOptions control how tracking, based on systemd transient scope, operates.
+type TrackingOptions struct {
+	// AllowSessionBus controls if CreateTransientScopeForTracking will
+	// consider using the session bus for making the request.
```

Comment:
I think I'd skip the other function name. The names can change, and then the comments do not get updated.

```suggestion
	// AllowSessionBus controls whether the request to set up tracking
	// can be sent over the session bus.
```

Reply:
We'll do this and the other change in a follow-up.